code/game/objects/items/food/bread.dm

```diff
@@ -0,0 +1,309 @@
+	name = "tofubread slice"
+	desc = "A slice of delicious tofubread."
+	icon_state = "tofubreadslice"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/vitamin = 1)
+	foodtypes = GRAIN | VEGETABLES
+
+/obj/item/food/bread/creamcheese
+	name = "cream cheese bread"
+	desc = "Yum yum yum!"
+	icon_state = "creamcheesebread"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 25, /datum/reagent/consumable/nutriment/vitamin = 10)
+	tastes = list("bread" = 10, "cheese" = 10)
+	foodtypes = GRAIN | DAIRY
+
+/obj/item/food/bread/creamcheese/Initialize()
+	. = ..()
+	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/breadslice/creamcheese, 5, 30)
+
+/obj/item/food/breadslice/creamcheese
+	name = "cream cheese bread slice"
+	desc = "A slice of yum!"
+	icon_state = "creamcheesebreadslice"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 4, /datum/reagent/consumable/nutriment/vitamin = 1)
+
+/obj/item/food/bread/mimana
+	name = "mimana bread"
+	desc = "Best eaten in silence."
+	icon_state = "mimanabread"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 25, /datum/reagent/toxin/mutetoxin = 5, /datum/reagent/consumable/nothing = 5, /datum/reagent/consumable/nutriment/vitamin = 10)
+	tastes = list("bread" = 10, "silence" = 10)
+	foodtypes = GRAIN | FRUIT
+
+/obj/item/food/bread/mimana/Initialize()
+	. = ..()
+	AddElement(/datum/element/processable, TOOL_KNIFE, /obj/item/food/breadslice/mimana, 5, 30)
+
+/obj/item/food/breadslice/mimana
+	name = "mimana bread slice"
+	desc = "A slice of silence!"
+	icon_state = "mimanabreadslice"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 2, /datum/reagent/toxin/mutetoxin = 1, /datum/reagent/consumable/nothing = 1, /datum/reagent/consumable/nutriment/vitamin = 1)
+	foodtypes = GRAIN | FRUIT
+
+/obj/item/food/breadslice/custom
+	name = "bread slice"
+	icon_state = "tofubreadslice"
+	foodtypes = GRAIN
+
+/obj/item/food/baguette
+	name = "baguette"
+	desc = "Bon appetit!"
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "baguette"
+	inhand_icon_state = "baguette"
+	worn_icon_state = "baguette"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 8, /datum/reagent/consumable/nutriment/vitamin = 3)
+	bite_consumption = 3
+	w_class = WEIGHT_CLASS_NORMAL
+	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT
+	attack_verb_continuous = list("touche's")
+	attack_verb_simple = list("touche")
+	tastes = list("bread" = 1)
+	foodtypes = GRAIN
+
+/obj/item/food/garlicbread
+	name = "garlic bread"
+	desc = "Alas, it is limited."
+	icon = 'icons/obj/food/burgerbread.dmi'
+	icon_state = "garlicbread"
+	inhand_icon_state = "garlicbread"
+	list_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/consumable/nutriment/vitamin = 6, /datum/reagent/consumable/garlic = 2)
+	bite_consumption = 3
+	tastes = list("bread" = 1, "garlic" = 1, "butter" = 1)
+	foodtypes = GRAIN
+
+/obj/item/food/deepfryholder
```

Comment:
bread.dm seems like a strange place for this to live

Reply:
fried food is _breaded_. i dunno that's my best guess.